check-before-deployment.md

```diff
@@ -128,6 +122,12 @@ swapoff -a && swapon -a
 sysctl -p
 ```
 
+**Note:**
+
+> - Executing `swapoff -a` and then `swapon -a` is to refresh swap by dumping data to memory and cleaning up swap. If you drop `swapon -a` and execute only `swapoff -a`, swap will be enabled again after you restart the system.
```

Comment:
```suggestion
> - Executing `swapoff -a` and then `swapon -a` is to refresh swap by dumping data to memory and cleaning up swap. If you drop the swappiness change and execute only `swapoff -a`, swap will be enabled again after you restart the system.
```

Reply:
Should we add `swapon -a`？That is, "If you drop the swappiness change and `swapon -a`, and execute only `swapoff -a`".